torch/csrc/jit/ir.h

```diff
@@ -858,6 +861,8 @@ struct Block {
   // having corner cases where the list is empty.
   Node * const output_;
   Node * const input_;
+  Node * const entry_world_;
```

Comment:
Are entry_world_ and exit_world_ used at the block level or just at the graph level?  If they are only used at the graph level, they should be defined on Graph and not Block.  Someone might call `entryWorld()` on the block, expecting it to be the loop-carried value that is the world token that is the input to a loop block body, but instead get an unused Value* instead.